### PR TITLE
🍰 feature/update_docker

### PR DESCRIPTION
**Pull request by Olderestin**
## What changed?     
Update docker compose up build, cause for the first build django don't wait for postgres conainer finish to create db and be ready to connections.

## Checklist before requesting a review     
- [x] I have performed a self-review of my code
- [x] Code is functional on my machine(Windows 10/Debian GNU/Linux 11 (bullseye))

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -10,14 +10,21 @@ services:
       - .env
     ports:
       - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
+      timeout: 5s
```
Comment (memhard) [inline docker-compose.yml]: why here timeout need if there interval with retries?

**Review by memhard — APPROVED** [review]
can Aprove, but first answer on my questions

**Review by Olderestin — COMMENTED** [review]
Update task.

**Review by memhard — APPROVED** [review]
LGTM 👍

**Pull request merged**